Makefile

```diff
@@ -41,7 +41,7 @@ vendor: ## Ensure the vendor directory is up to date.
 
 .PHONY: lint
 lint: ## Run golangci-lint over the codebase.
-	${GOLANGCI_LINT} run ./... --timeout 5m -v
+	${GOLANGCI_LINT} run ./... --timeout 5m -v --fix
```

Comment:
I wouldn't want this checked in, as it will affect the CI check.

Can you make an alternative `lint-fix` target that runs the same lint task but with the fix option please?